deploy/central/tasks-chart/templates/task-ods-build-go.yaml

```diff
@@ -50,6 +50,10 @@ spec:
         without leading `./` and trailing `/`.
       type: string
       default: "."
+    - name: cache-dependencies
+      description: Whether to cache dependencies in the PVC.
+      type: string
+      default: "false"
```

Comment:
I am wondering if we shouldn't turn this on by default. I guess the downside of caching dependencies is disk usage - otherwise I do not see a reason not to use it?!

Reply:
For go this makes sense to me, but I am not sure about other languages. Would this make a difference?